production/helm/loki/templates/gateway/service-gateway.yaml

```
@@ -12,6 +12,7 @@ metadata:
     {{- with .Values.gateway.service.labels }}
     {{- toYaml . | nindent 4}}
     {{- end }}
+    prometheus.io/service-monitor: "false"
```

it must be added only if NGINX is used as gateway. 
```suggestion
    {{- if not (and .Values.enterprise.enabled .Values.enterprise.gelGateway) }}
    prometheus.io/service-monitor: "false"
    {{- end }}
```